sklearn/neural_network/_multilayer_perceptron.py

```diff
@@ -1037,8 +1043,7 @@ def fit(self, X, y):
         -------
         self : returns a trained MLP model.
         """
-        return self._fit(X, y, incremental=(self.warm_start and
-                                            hasattr(self, "classes_")))
+        return self._fit(X, y, incremental=False)
```

Comment:
We can actually remove the `fit` function because it is the same as the base class then.

Reply:
Indeed indeed, good point! I will update the PR.

Do you prefer a push --force with the initial commit removing the function, a a commit on top of the current status?

Reply:
We prefer to commit on top because it helps us keep the context of the comments on github.

Reply:
Thanks @thomasjpfan
I'll send new commits after the weekend (currently sweating the hell out of it climbing).

Reply:
Done that with the last set of commits pushed.